### ISSUE TITLE
Implement shift opcodes

### ISSUE THREAD

**Issue by Prabhat1308** [tonk-gg/dappicom#40]
Implement ASL , LSR , ROL , ROR opcodes

**Comment by fishonamos** [comment]
Hi @Prabhat1308 , can you provide more context for this issue?

**Comment by goblinoats** [comment]
Please take a look at #6, opening an issue with listed opcodes is a way to describe the intent to implement those opcodes. Correct me if I'm wrong @Prabhat1308.

**Comment by Prabhat1308** [comment]
Yes exactly . @fishonamos  you can refer to #6 for instructions on how to implement opcode . I will suggest you to look at the  opcodes already implemented to get acquainted with any style patterns of writing opcodes.
